Dockerfile

```diff
@@ -30,17 +29,27 @@ ENV RUSTC_FLAGS='-C target-cpu=x86-64'
 RUN --mount=type=cache,target=/tmp/target \
     --mount=type=cache,target=/usr/local/cargo/git \
     --mount=type=cache,target=/usr/local/cargo/registry \
+    echo "$PATH" && \
+    ls /usr/local/cargo && \
```

Comment:
Is this a debug-only information? Should we drop it?

Reply:
Yup